pkg/processor/runtime/python/runtime.go

```diff
@@ -174,3 +179,27 @@ func (py *python) getPythonExePath() (string, error) {
 func (py *python) GetEventEncoder(writer io.Writer) rpc.EventEncoder {
 	return rpc.NewEventMsgPackEncoder(py.Logger, writer)
 }
+
+func (py *python) resolveDecodeEvents() bool {
```

Comment:
```suggestion
func (py *python) resolveEventStringDecoding() bool {
```
Would that be more accurate? or is there other types decoded here?

Reply:
no other types to decode